docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,123 @@
+| ------ | ------------------------------------------------------------------------------------- |
+| `Name` | The data source name. This is how you refer to the data source in panels and queries. |
+
+### HTTP Settings
+
+| Name     | Description                                                                                               | Value                    |
+| -------- | --------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `URL`    | The IP address/hostname and optional port of your Elasticsearch instance.                                 | `http://localhost:12200` |
+| `Access` | whether to connect to your Elasticsearch instance from the browser or through Grafana's backend (Server). | `Server (default)`       |
+
+### Elasticsearch details
+
+| Name              | Description                                                                | Value                  |
+| ----------------- | -------------------------------------------------------------------------- | ---------------------- |
+| `Index name`      | The index pattern used to identify indices you want to query with grafana. | `[metrics-]YYYY.MM.DD` |
+| `Pattern`         | The time pattern for which the above index will be be interpolated.        | `Daily`                |
+| `Time field name` | The name of the field that identiefies the timestamp of your documents.    | `@timestamp`           |
+| `Version`         | The Elasticsearch version you are running.                                 | `7.0+`                 |
+
+Once you click **Save & Test** the following message will appear, confirming that Grafana is able to communicate with your Elasticsearch instance:
+
+![Elasticsearch confirmation message](/img/docs/getting-started/elasticsearch/confirmation-7-4.png)
+
+## Step 4. Create your first Elasticsearch backed dashboard
+
+1. In the Grafana side menu, hover your cursor over the **Create** (plus) icon and then click **Dashboard**.
+1. Click the **Add new panel** button.
+
+If you made Elasticsearch your default data source by toggling `default` on when creating it, it should be already selected:
+
+![Elasticsearch panel edit](/img/docs/getting-started/elasticsearch/panel-edit-7-4.png)
+
+Otherwise you can select it from the data source picker:
+
+![data source picker](/img/docs/getting-started/elasticsearch/datasource-picker-7-4.png)
+
+You can now start editing your query to get data out of your Elasticsearch instance.
+The default query will graph the number of received documents in our index over the specidied period of time:
+
+![Default query](/img/docs/getting-started/elasticsearch/default-query-7-4.png)
+
+## Next steps
+
+Now that you have your Elasticsearch powered Grafana dashboard you can start [exploring](/docs/grafana/latest/explore/) different ways of querying your Elasticsearch index to get the most out of your data.
```

Comment:
```suggestion
Now that you have your Elasticsearch powered Grafana dashboard you can start [exploring](/docs/grafana/latest/explore/_index.md) different ways of querying your Elasticsearch index to get the most out of your data.
```

Reply:
This doesn't work when building the website locally.

Reply:
Update from Master. This was a recent change, and it is correct.